src/lib/y2storage/boot_requirements_strategies/zipl.rb

```diff
@@ -28,25 +28,29 @@ module BootRequirementsStrategies
     # Strategy to calculate boot requirements in systems using ZIPL
     class ZIPL < Base
       def needed_partitions(target)
-        raise Error unless supported_root_disk?
-        [zipl_partition(target)]
+        raise Error unless supported_boot_disk?
+        zipl_partition_missing? ? [zipl_partition(target)] : []
       end
 
     protected
 
-      def supported_root_disk?
-        return false unless root_disk
-        if root_disk.is?(:dasd)
-          return false if root_disk.dasd_type.is?(:fba)
-          return false if root_disk.dasd_format.is?(:ldl)
+      def supported_boot_disk?
+        return false unless boot_disk
+        if boot_disk.is?(:dasd)
+          return false if boot_disk.dasd_type.is?(:fba)
+          return false if boot_disk.dasd_format.is?(:ldl)
```

Comment:
Once again, I consider this to be quite out of the scope of this pull request. Maybe we will take a look, but I cannot promise it. :smile: